MapboxSearch/ui/src/main/java/com/mapbox/search/ui/adapter/autocomplete/PlaceAutocompleteItemsCreator.kt

```diff
@@ -34,7 +34,7 @@ internal class PlaceAutocompleteItemsCreator(
             val distance = if (suggestion.distanceMeters != null) {
                 suggestion.distanceMeters
             } else {
-                locationEngine.lastKnownLocation(context).value?.distanceTo(suggestion.coordinate)
+                suggestion.coordinate?.let { locationEngine.lastKnownLocation(context).value?.distanceTo(it) }
```

Comment:
Please add comments here to describe the behavior

Reply:
Done